src/tests/cmocka/test_iobuf.c

```diff
@@ -50,6 +50,7 @@ static void copy_heap_to_file(int out, int in, off_t start, off_t end)
     assert_int_equal(pos, start);
 
     while (size > 0) {
+        explicit_bzero(buffer, sizeof(buffer));
```

Comment:
Please use `sss_erase_mem_securely()` as `explicit_bzero()` might be unavailable.

Reply:
Done.